src/media/css/footer.styl

```diff
@@ -45,6 +45,11 @@ $support-nav-height = 55px;
             display: block;
         }
     }
+
+    .page-footer-text {
+         color:$action-neutral-hover;
```

Comment:
`$action-neutral` would be better since it would match the normal page text color